src/components/TaskCardButtonsContainer/TaskCardButtonsContainer.jsx

```diff
@@ -19,7 +21,10 @@ function TaskCardButtonsContainer({
       {isTaskDone ? (
         <p>{`Completed in ${numberOfDays} ${dayStr}`}</p>
       ) : (
-        <DoneButton onDone={onTaskDone} />
+        <>
+          <EditButton onEdit={onEditTask} />
+          <DoneButton onDone={onTaskDone} />
+        </>
       )}
     </>
```

Comment:
Why are we using nested react Fragment?

Reply:
@rsa-wtag as there is a conditional rendering going on from line 23, these to elements need to have a parent wrapper. Otherwise there will be an error.